test/e2e/packages/ubuntu-20.04/n4c16/test01-systemd/code.var.sh

```diff
@@ -0,0 +1,10 @@
+# Clear cri-resmgr output from previous runs.
```

Comment:
This starts to look like we could have `code.var.sh` on `packages` level and add `export_var_files "$TESTS_ROOT_DIR"` to `run_tests.sh`...

Reply:
Yes, I was already long considering generalizing the current concept and changing `run_tests.sh` so that before each call to `run.sh` it would climb back the directory tree from the test directory to the test root, collect all `.sh` files, and then source and export them in reverse order (IOW top-to-bottom) that would give us basically a kind of overrideable inheritance in the test subtree.

Reply:
Yep, I'll add it. Probably makes sense after getting this patch in.

(I'm not great fan hardcoded number and naming of policy-topology-test levels either. That could use some refactoring at the same time. Now we already know better how the tests tend to be structured.)

Currently we have `*.var.*` files that contain only variable value (without eval/echo/source etc templating). `${FILENAME%.var*}` is the name of the variable. Overrideable inheritance works for these already - except for that exporting vars from the topmost directory is missing.

Would you like to have also `source`able `*.sh` files in the tree?